scene/gui/text_edit.cpp

```diff
@@ -4825,6 +4825,9 @@ void TextEdit::deselect() {
 
 void TextEdit::select(int p_from_line, int p_from_column, int p_to_line, int p_to_column) {
 
+	ERR_FAIL_COND(p_to_line < 0)
+	ERR_FAIL_COND(p_from_line < 0)
```

Comment:
I'm surprised this compiles, but `ERR_FAIL_COND()` macros should end with a semicolon.

Reply:
I wonder why the `_column` parameters have checks resetting them to 0 below, but not the `_line` parameters though. CC @Paulb23

Reply:
I don't think there's any particular reason, resetting them to `0` would be fine.

Reply:
Bump.

Reply:
Done